manifests/install.pp

```diff
@@ -33,12 +33,18 @@
     require => Exec["pyenv::checkout ${user}"],
   }
 
-  exec { "pyenv::shrc ${user}":
-    command => "echo 'source ${pyenvrc}' >> ${shrc}",
-    user    => $user,
+  file { $shrc:
```

Comment:
This is causing issues (https://travis-ci.org/abloomston/puppet-pyenv/jobs/194917442), and I'm not sure it's actually required. I've removed it from this branch while investigating.